### ISSUE TITLE
Repeater component accessibility issues 

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1071]
The [repeater component](https://pulsar.docs.jadu.net/forms/repeater) contains various accessibility issues.

1. There are inputs without labels within `repeater__saved-data`. Could these be swapped for `type="hidden"`?
2. When the repeater form expands (either by add another or edit), focus should be set to the first input.
3. When the repeater form is closed, focus should be set to the "Add Another" button.
4. The "Add another" button is visually disabled when the form is open but can still be focussed. The `disabled` attribute should also be added.
5. The table should have a `caption` which matches the `label`.
6. There are various mark up validation failures relating to the table.
